data/human/variants.txt

```diff
@@ -2613,6 +2613,92 @@ ship "Scout" "Scout (Speedy)"
 
 
 
+ship "Scrapper" "Scrapper (Minimal)"
+	outfits
+		"Energy Blaster"
+		
+		"nGVF-AA Fuel Cell"
```

Comment:
```suggestion
		"KP-6 Photovoltaic Panel"
```

With recent changes to power systems, it is actually feasible to do a battery only build. I'm not sure our AI is smart enough to realize it needs to run away when the battery gets low, though. I think our fighters are smart enough to head back to their carrier, but a standalone ship? Not sure. As such, the next best thing to being a pure-battery setup for a minimalist version would be a single solar panel.

Reply:
Even if the AI isnt smart that'd fit the beginner pirate/ship out of nothing and doesnt function properly perfectly

Reply:
At first I was a little worried they'd be too weak to last more than a few seconds, tested and seems like they'd last long enough to disable a Sparrow (assuming the Scrapper didn't die first) with spare to fly for a bit so should be enough for them to jump in, disable a Shuttle, plunder, and leave. I'll apply this suggestion.